airtable/airtable_to_mailing_list.py

```diff
@@ -109,25 +109,31 @@ def sync_airtable_to_mailing_list():
         valid_chapters_to_add = []
         for c in m.chapters_to_add:
             if c not in chapter_to_mailing_list:
+                print "Not adding member {} to chapter {} because it does not have a mailing list.".format(m.email, c)
```

Comment:
If this is being run by cron then I believe anything printed will be sent in an email to us. Which means it might be a good idea to only print stuff if something went wrong that a human needs to take a look at.